examples/pom.xml

```diff
@@ -276,4 +276,60 @@
 			</build>
 		</profile>
     </profiles>
+    
+    <repositories>
```

Comment:
Product stuff should not leak into the project. I assume that this ends up in wildfly/docs/examples, does it?
In that location, those examples should work without any more repos than we already define.

For this to work in prod, it should also work already with the repos prod already defines